public/achievementList.php

```diff
@@ -32,6 +32,9 @@
 $sortBy = (int) request()->input('s', 17);
 $achData = getAchievementsList($user, $sortBy, $params, $count, $offset, $flags, $dev);
 
+// Is the user looking at their own achievements list?
+$isOwnEarnedAchievementsList = $user !== null;
```

Comment:
The only way $user would be null is if they're not logged in, and I don't think including the Created/Modified timestamps for that scenario gets us anything. Recommend removing them there too.

Reply:
Upon closer inspection it looks like I screwed up a bit of the conditional logic in my first pass of this PR. Hopefully it is buttoned up now.

The intent is for all achievement lists that are not the user's "My Unlocked Achievements" to be functioning as they do in prod (we can of course change this if you'd like). "Awarded" now only appears when the `p` query param is set to `1`.